testdata/p4_16_errors/structured-annotation-e.p4

```diff
@@ -0,0 +1,4 @@
+@IllegalMixing[key=4, 5]
```

Comment:
There is no expected output file for this test program?

Reply:
Error programs may not have outputs, depending on where the compiler exits.

Reply:
This test program does have an error output, when I try it out with your changes on my local dev machine:
```
$ ~/forks/p4c/build/p4test structured-annotation-e.p4 
structured-annotation-e.p4(1):syntax error, unexpected INTEGER
@IllegalMixing[key=4, 5
                      ^
error: 1 errors encountered, aborting compilation
```